app/src/main/res/layout/navdrawer.xml

```diff
@@ -16,55 +16,12 @@
   ~ limitations under the License.
 -->
 
-<org.gdg.frisbee.android.widget.ScrimInsetsFrameLayout android:id="@+id/navdrawer"
+<android.support.design.widget.NavigationView
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:layout_width="@dimen/navdrawer_width"
+  android:id="@+id/nav_view"
+  android:layout_width="wrap_content"
   android:layout_height="match_parent"
   android:layout_gravity="start"
   android:fitsSystemWindows="true"
-  android:background="@color/navdrawer_background"
-  app:insetForeground="@color/navdrawer_inset_foreground">
-
-  <LinearLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_gravity="start"
-    android:clickable="true"
```

Comment:
@tasomaniac I was that person who had reported this issue :) I have already checked this case before committing the code!